test/infrastructure/docker/e2e/ci-e2e.conf

```diff
@@ -1,13 +1,13 @@
 ---
 images:
-- name: gcr.io/k8s-staging-capi-docker/capd-manager-amd64:dev
+- name: ci-registry/docker-provider-manager-amd64:ci
```

Comment:
Do we want to do this and break the use of the make target directly?

Reply:
I'm not done yet, once this actually passes CI then i'm going to include the default e2e config that is meant to be used locally.